test/witan/app/forecast_test.clj

```diff
@@ -0,0 +1,31 @@
+(ns witan.app.forecast-test
```

Comment:
ok, good to know.
the stubbing here is purely to keep from having to have a test keyspace with all the setup that implies - also that also comes with its own pitfalls (as we found out) since we don't test what's going on under the hood.